main/src/main/resources/org/clulab/numeric/HOLIDAY.tsv

```diff
@@ -0,0 +1,49 @@
+#
+# List of holidays in the United States extracted from the Jollyday api.
+# This list can be obtained with the following snippet:
+#
+#   import de.jollyday.{HolidayManager, HolidayCalendar, ManagerParameters}
+#   import collection.JavaConverters._
+#   val holidayManager = HolidayManager.getInstance(ManagerParameters.create(HolidayCalendar.UNITED_STATES))
+#   holidayManager.getCalendarHierarchy.getChildren.keySet.asScala.
+#                   map(holidayManager.getHolidays(2021, _).asScala).flatten.map(_.getDescription)
+#
+#
+Lincoln's Birthday
+Easter Monday
+Washington's Birthday
+Labour day
+César Chávez's Birthday
+Seward's Day
+Christmas Eve
+Mardi Gras
+Battle of Bennington
+Arbor Day
+West Virginia Day
+Election day
+Confederate Memorial Day
+Statehood day
+Truman Day
+Christmas
+Presidents Day
+Prince Jonah Kuhio Kalanianaole Day
+Independence Day
+New Years Eve
+Jefferson Davis Day
+Lee-Jackson Day
+Patriots' Day
+Alaska Day
+Victory Day
+New Year
+Veterans Day
+Columbus Day
+undefined
+Thanksgiving Day
+Martin Luther King, Jr. Day
```

Comment:
AKA MLK Day?  Is it worth adding alternatives?

Reply:
We could add alternatives and use comments as canonical names. That could look to something like:
```
Martin Luther King, Jr. Day // Martin Luther King, Jr. Day
MLK Day // Martin Luther King, Jr. Day
```
What do you think?